src/BreathOfTheWild/Mods/DayLength/rules.txt

```diff
@@ -29,99 +31,169 @@ name = 15 seconds
 category = Day Length
 condition = ($timeCycleMode == 1)
 $timeMultiplier = 96
-$cloudMultiplier = 96
 
 [Preset]
 name = 30 seconds
 category = Day Length
 condition = ($timeCycleMode == 1)
 $timeMultiplier = 48
-$cloudMultiplier = 48
 
 [Preset]
-name = 1 minutes
+name = 1 minute
 category = Day Length
 condition = ($timeCycleMode == 1)
 $timeMultiplier = 24
-$cloudMultiplier = 24
 
 [Preset]
 name = 6 minutes
 category = Day Length
 condition = ($timeCycleMode == 1)
 $timeMultiplier = 4
-$cloudMultiplier = 4
 
 [Preset]
 name = 12 minutes
 category = Day Length
 condition = ($timeCycleMode == 1)
 $timeMultiplier = 2
-$cloudMultiplier = 2
 
 [Preset]
 name = 24 minutes (Default)
 category = Day Length
 condition = ($timeCycleMode == 1)
 default = 1
 $timeMultiplier = 1.0
-$cloudMultiplier = 1.0
 
 [Preset]
 name = 36 minutes
 category = Day Length
 condition = ($timeCycleMode == 1)
 $timeMultiplier = 0.75
-$cloudMultiplier = 0.75
 
 [Preset]
 name = 48 minutes
 category = Day Length
 condition = ($timeCycleMode == 1)
 $timeMultiplier = 0.5
-$cloudMultiplier = 0.5
 
 [Preset]
 name = 72 minutes
 category = Day Length
 condition = ($timeCycleMode == 1)
 $timeMultiplier = (1/3)
-$cloudMultiplier = (1/3)
 
 [Preset]
 name = 96 minutes
 category = Day Length
 condition = ($timeCycleMode == 1)
 $timeMultiplier = 0.25
-$cloudMultiplier = 0.25
 
 [Preset]
 name = 2 hours
 category = Day Length
 condition = ($timeCycleMode == 1)
 $timeMultiplier = 0.2
-$cloudMultiplier = 0.2
 
 [Preset]
 name = 4 hours
 category = Day Length
 condition = ($timeCycleMode == 1)
 $timeMultiplier = 0.1
-$cloudMultiplier = 0.1
 
 [Preset]
 name = 12 hours
 category = Day Length
 condition = ($timeCycleMode == 1)
-$timeMultiplier = (1/3)/10
-$cloudMultiplier = (1/3)/10
+$timeMultiplier = (1/30)
 
 [Preset]
 name = 24 hours
 category = Day Length
 condition = ($timeCycleMode == 1)
-$timeMultiplier = (1/6)/10
-$cloudMultiplier = (1/6)/10
+$timeMultiplier = (1/60)
+
+
+[Preset]
+name = Normal
+category = Cloud Speed
+condition = ($timeCycleMode == 1)
+$cloudMultiplier = 1.0
+
+[Preset]
+name = Scaled with day length
+category = Cloud Speed
+condition = ($timeCycleMode == 1)
+default = 1
+$cloudMultiplier = 0
+
+
+[Preset]
+name = 5 minutes
+category = Clock Interval
+condition = ($timeCycleMode == 1)
+default = 1
+$clockAdjust:int = 1
+
+[Preset]
+name = 1 minute
```

Comment:
```suggestion
name = Increment Clock By 1 Minute
```
Could go with every minute too, but I think it's easier to parse if you use the same formatting for both. Just my thoughts.

Reply:
I was unsure about what to put for a lot of that label text.  Your suggestions all seem good.
The other things I was unsure about was the exact values for the blood moon presets or if I should add other cloud speed presets.

Reply:
I think with the amount of presets and types of presets I'd generally just go with what makes sense to you and you see someone using.